x-pack/legacy/plugins/ml/public/datavisualizer/breadcrumbs.ts

```diff
@@ -5,10 +5,10 @@
  */
 
 // @ts-ignore
-import { ML_BREADCRUMB } from '../breadcrumbs';
+import { ML_BREADCRUMB, ANOMALY_DETECTION_BREADCRUMB } from '../breadcrumbs';
```

Comment:
Is there a reason why the Anomaly Detection breadcrumb is in here? 

![image](https://user-images.githubusercontent.com/7405507/64692002-f35cd200-d48b-11e9-9b07-83d17c0731cf.png)

Should just be Machine Learning / Data Visualizer I think. Or even Machine Learning / Data Visualizer / Index based?

Reply:
It shouldn't be. Fixed in: 047c7a71b68138b953c87df8e145cb83d447921c